sys/psa_crypto/include/psa_hashes.h

```diff
@@ -341,6 +341,82 @@ psa_status_t psa_hashes_sha512_256_finish(psa_hashes_sha512_256_ctx_t *ctx,
                                       size_t *hash_length);
 #endif /* MODULE_PSA_HASH_SHA_512_256 */
 
+#if IS_USED(MODULE_PSA_HASH_SHA_3) || defined(DOXYGEN)
+/**
+ * @brief   Low level wrapper function to call a driver for a SHA3-256 hash setup
+ *          See @ref psa_hash_setup()
+ *
+ * @param ctx
+ * @return psa_status_t
+ */
+psa_status_t psa_hashes_sha3_256_setup(psa_hashes_sha3_ctx_t *ctx);
```

Comment:
Would it make sense to have different psa modules for SHA3-256/384/512 or is the increase in code size if including all three per default negligible?

Reply:
@mguetschow The increase in code-size is indeed negligible since they are all based on the same keccak-code (which is the biggest part of the underlying code), which needs to be compiled anyway with every variant of the sha3 algorithm.

Reply:
Okay, then it sounds reasonable to me to keep it as is. But maybe @Einhornhool has a different opinion and would argue in favor of consistency with the SHA-2 PSA modules?

Reply:
> I think separation is only needed once we get a hardware accelerator that only supports a subset of them.

but that would be an API change then

Reply:
I considered this and I agree. This would be an API change, which makes it more prone to errors.
Also it would require people who add drivers in the future to not only add wrappers, but also to change the module structure.
That makes it less friendly to develop.

I'm sorry for changing my mind now, but I think it would be better to keep the structure consistent with the other modules :)